compiler/src/iree/compiler/Pipelines/Options.cpp

```diff
@@ -245,6 +245,18 @@ void PreprocessingOptions::bindOptions(OptionsBinder &binder) {
       llvm::cl::desc(
           "File name of a transform dialect spec to use for preprocessing"),
       llvm::cl::cat(category));
+
+  binder.opt<TransposeMatmulInput>(
+      "iree-preprocessing-enable-transpose-matmul",
```

Comment:
nit: I learned that `enable` is redundant in the flag -- when flags are boolean what decides if something is enabled or not is the value, not the presence of the flag, and it's just an extra word.

Reply:
I initially tried that but it's the same as the flag in Passes.td and I hit a "flag is registered more than once" runtime error. That's why I added "enable", seems common in other flags as well. Open to suggestions on alternatives.

Reply:
oh I see, I did not notice that. How about add `-pass` suffix to the flag defined in `Passes.td` file?